src/app/tests/yaml/onoff.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: OnOff Tests
+
+config:
+    cluster: "OnOff"
+    endpoint: 1
+
+tests:
+    - label: "Check on/off attribute value"
+      type: "readAttribute"
+      name: "onOff"
```

Comment:
both label and name seem confusing. Generally I expect the name to be what it runs, but here I assume it is cluster.

Should name be `cluster` or `command` instead?